tests/forms/FormFieldTest.php

```diff
@@ -159,6 +159,9 @@ public function testReadonly() {
 		$this->assertContains('readonly="readonly"', $field->getAttributesHTML());
 		$field->setReadonly(false);
 		$this->assertNotContains('readonly="readonly"', $field->getAttributesHTML());
+		$this->assertInstanceOf('FormField', $field->performReadonlyTransformation());
+		eval("class FormField_Readonly extends FormField {}");
```

Comment:
`eval()` is the only way I found to dynamically define a class. If this is a problem I can just drop the test with FormField_Readonly defined.